float32 grasp_x_cm
float32 grasp_y_cm
float32 grasp_theta


